### PR TITLE
fix(server): fix custom plan updates (cpu/mem)

### PR DESCRIPTION
**Pull request by peknur**

### CODE REVIEW

File: upcloud/resource_upcloud_server_test.go [hunk 1]
```diff
@@ -13,6 +13,59 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
+func TestUpcloudServer_custom(t *testing.T) {
```
Comment (ka-myl) [inline upcloud/resource_upcloud_server_test.go:16]: Would name it `customPlan` or something like that, as users can also have custom server images :)

**Review by ka-myl — APPROVED** [review]
LGTM :+1:

File: upcloud/resource_upcloud_server_test.go (hunk 1 above)
Comment (peknur) [reply upcloud/resource_upcloud_server_test.go:16]: Good point! I will rename it.

**Review by Darep — APPROVED** [review]
LGTM! Tested locally and everything looked good! :) Fix works and other things still work, too :D

**Pull request merged**